lib/bummr/updater.rb

```diff
@@ -36,7 +36,9 @@ def update_gem(gem, index)
         log("#{gem[:name]} not updated from #{gem[:installed]} to latest: #{gem[:newest]}")
       end
 
-      git.add("Gemfile Gemfile.lock vendor/cache")
+      git.add("Gemfile")
+      git.add("Gemfile.lock")
+      git.add("vendor/cache")
```

Comment:
The issue was that pathspec for `vendor/cache` was nil and throwing an error. Adding separately fixes this somehow.